packages/analytics/src/integrations/index.ts

```diff
@@ -1,2 +1,6 @@
+import { formatPageEvent, formatTrackEvent } from './Omnitracking';
+
+export const OmnitrackingHelpers = { formatPageEvent, formatTrackEvent };
```

Comment:
Why is this being exported here?

Reply:
Forter uses FormatTrackEvent when it trigger, internally, an omnitracking event.

Reply:
Right, when I searched for it in VSCode somehow it was not finding it but I can see it now.
In this case, I would suggest exposing these methods as static inside the `Omnitracking` integration class, then consumers who needed it would use them like this:

```js
const formattedEvent = Omnitracking.formatPageEvent(event)
```

And then you can get rid of these exports as they "pollute" the integrations export with other things that are not integrations.